config/jacoco.gradle

```diff
@@ -22,27 +29,38 @@ task jacocoTestReport(type: JacocoReport, dependsOn: "testStagingDebugUnitTest")
                          'android/**/*.*']
 
     def packagesExcluded = ['com/nimbl3/di/**',
+                            'com/nimbl3/ui/**/di/**',
                             'com/nimbl3/extension/**',
                             'com/nimbl3/**/extension/**']
-    // def filesExcluded = [ ]
-    // def fileFilter = fileGenerated + packagesExcluded + filesExcluded
+
     def fileFilter = fileGenerated + packagesExcluded
 
-    classDirectories = fileTree(
-        dir: "$buildDir/intermediates/javac/stagingDebug/compileStagingDebugJavaWithJavac/classes",
+    classDirectories.from = fileTree(
+        dir: "$buildDir/intermediates/javac/stagingDebug/classes",
+        excludes: fileFilter
+    ) + fileTree(
+        dir: "$buildDir/intermediates/javac/stagingDebug/classes",
+        excludes: fileFilter
```

Comment:
I see a duplication here? 🤔

Reply:
Yes @Tuubz we can use one, if we use $buildDir instead project.rootDir, I will update.